api/src/test/java/org/pmiops/workbench/workspaces/WorkspacesControllerTest.java

```diff
@@ -3145,4 +3162,102 @@ public void copyNotebook_validateActiveBilling() {
+        cohortsController
+            .createCohort(
+                workspace.getNamespace(), workspace.getId(), createDefaultCohort("cohort"))
+            .getBody();
+    stubBigQueryCohortCalls();
+    CohortReview cohortReview =
+        cohortReviewController
+            .createCohortReview(
+                workspace.getNamespace(),
+                workspace.getId(),
+                cohort.getId(),
+                cdrVersion.getCdrVersionId(),
+                new CreateReviewRequest().size(1))
+            .getBody();
+    ConceptSet conceptSet =
+        conceptSetsController
+            .createConceptSet(
+                workspace.getNamespace(),
+                workspace.getId(),
+                new CreateConceptSetRequest()
+                    .conceptSet(
+                        new ConceptSet().name("cs1").description("d1").domain(Domain.CONDITION))
+                    .addAddedIdsItem(CONCEPT_1.getConceptId()))
+            .getBody();
+    DataSet dataSet =
+        dataSetController
+            .createDataSet(
+                workspace.getNamespace(),
+                workspace.getId(),
+                new DataSetRequest()
+                    .prePackagedConceptSet(PrePackagedConceptSetEnum.NONE)
+                    .addConceptSetIdsItem(conceptSet.getId())
+                    .addCohortIdsItem(cohort.getId())
+                    .name("dataset")
+                    .domainValuePairs(
+                        ImmutableList.of(
+                            new DomainValuePair().value("VALUE").domain(Domain.CONDITION))))
+            .getBody();
+
+    WorkspaceResourceResponse workspaceResourceResponse =
+        workspacesController.getCdrSelectors(workspace.getNamespace(), workspace.getId()).getBody();
+    assertThat(workspaceResourceResponse.size()).isEqualTo(4);
+    List<WorkspaceResource> cohorts =
+        workspaceResourceResponse.stream()
+            .filter(workspaceResource -> workspaceResource.getCohort() != null)
+            .collect(Collectors.toList());
+    List<WorkspaceResource> cohortReviews =
+        workspaceResourceResponse.stream()
+            .filter(workspaceResource -> workspaceResource.getCohortReview() != null)
+            .collect(Collectors.toList());
+    List<WorkspaceResource> conceptSets =
+        workspaceResourceResponse.stream()
+            .filter(workspaceResource -> workspaceResource.getConceptSet() != null)
+            .collect(Collectors.toList());
+    List<WorkspaceResource> dataSets =
+        workspaceResourceResponse.stream()
+            .filter(workspaceResource -> workspaceResource.getDataSet() != null)
+            .collect(Collectors.toList());
+    assertThat(cohorts.size()).isEqualTo(1);
+    assertThat(cohorts.get(0).getCohort()).isEqualTo(cohort);
+    assertThat(cohortReviews.size()).isEqualTo(1);
+    compareCohortReviewFields(cohortReviews.get(0).getCohortReview(), cohortReview);
+    assertThat(conceptSets.size()).isEqualTo(1);
```

Comment:
please use `assertThat(conceptSets).hasSize(1);` here and wherever you're checking a collection's size